### ISSUE TITLE
Results: include link to specific worldpop dataset(s)

### ISSUE THREAD

**Issue by anandthakker** [developmentseed/worldpop#32]
On returning results for a polygon, include a link to the worldpop.co.uk page to download the original data.

**Comment by ianschuler** [comment]
Yeah, maybe a feature enhancement for a later phase, but it would be cool to generate a citation that includes all the dataset/year combinations for your polygon. With a nice copy functionality. Yumm.

**Comment by anandthakker** [comment]
Whoops, wrong issue.
